_data/tutorials.yml

```diff
@@ -111,6 +111,14 @@ code.org:
   url: https://code.org/learn
   categories:
   - blocks
+  
+  scratch-maus:
+  title: Programmieren mit der Maus
```

Comment:
wenn der Titel "de:" am Anfang hat, dann wird er nur in deutschen Tutorials angezeigt.